### PR TITLE
refactored monthly carousels out of template

### PR DESCRIPTION
**Pull request by mekarpeles**
Refactors homepage to remove monthly logic out of template. These queries should next be replaced with `List` queries made through `QueryCarousel` macro and removed from `home.py` entirely.

### Testing

Homepage works on testing & shows home carousels after cache is cleared

Should result in no visual changes

### Stakeholders

@cdrini @Sabreen-Parveen @seabelis

### CODE REVIEW

**Comment by mekarpeles** [comment]
@seabelis can we work together to turn these IDs into Lists on the https://openlibrary.org/people/booklists/lists account?

I can share s3 keys for this account and we have a javascript mechanism for adding in bulk:
https://github.com/internetarchive/openlibrary/wiki/Creating-Community-Lists

**Comment by mekarpeles** [comment]
Since https://github.com/internetarchive/openlibrary/pull/5292#issuecomment-859148859 is complete, once #5299 is merged, this PR can likely actually be closed

File: openlibrary/templates/home/index.html [hunk 1]
```diff
@@ -15,43 +15,13 @@
...
-$ monthly_reads = MONTHLY_READS.get(today().month)
-
 <div id="contentBody">
   $:render_template("home/categories", test=test)
 
   $:render_template("books/custom_carousel", books=readonline_carousel(), title=_('Classic Books'), url="/read", key="public_domain", test=test)
 
   $if monthly_reads and not test:
-    $:macros.QueryCarousel(query=monthly_reads['query'], title=monthly_reads['title'], key="monthly_reads", url=monthly_reads['url'], sort='random.hourly')
+    $:macros.QueryCarousel(query=monthly_reads['query'], title=_(monthly_reads['title']), key="monthly_reads", url=monthly_reads['url'], sort='random.hourly')
```
Comment (cdrini) [inline openlibrary/templates/home/index.html:24]: This won't work, `_` will only work with compile time static values.

File: openlibrary/plugins/openlibrary/home.py [hunk 2]
```diff
@@ -29,19 +30,237 @@
     )
 }
 
+MONTHLY_READS = {
+    2: {
+        "title": "Books for February",
```
Comment (cdrini) [inline openlibrary/plugins/openlibrary/home.py:35]:
```suggestion
        "title": _("Books for February"),
```

**Review by cdrini — APPROVED** [review]
Lgtm! Tested on testing.

**Pull request merged**